packages/web3-common/src/eth_execution_api.ts

```diff
@@ -140,7 +140,7 @@ export interface Log {
 	readonly blockNumber?: Uint | null;
 	readonly address?: Address;
 	readonly data?: HexStringBytes;
-	readonly topics?: Topic[];
+	readonly topics?: null | Topic | Topic[];
```

Comment:
The [issue](https://app.zenhub.com/workspaces/web3js-community-issues-management-612fd4e133c88b001205e29a/issues/chainsafe/web3.js/4784) suggests to : Update [topics](https://github.com/ChainSafe/web3.js/blob/cc6e15b1de5312b5a665606c1403958073807720/packages/web3-common/src/eth_execution_api.ts#L139) type in Log to include null and Topic (so final type would be null | Topics | Topics[]

However, as per [execution api def](https://github.com/ethereum/execution-apis/blob/68cf607f01bd41aecd0aaafd2bfc01928e188cc2/src/schemas/filter.json#L66), [Log Filter Topic](https://github.com/ChainSafe/web3.js/blob/cc6e15b1de5312b5a665606c1403958073807720/packages/web3-utils/src/types.ts#L25) can be bytes32, array of bytes32 or  null. 

So, it wouldn't be better, the initial Topic type to be: HexString32Bytes | HexString32Bytes[] | null and [topics](https://github.com/ChainSafe/web3.js/blob/cc6e15b1de5312b5a665606c1403958073807720/packages/web3-common/src/eth_execution_api.ts#L139) just stay Topic?

Reply:
The `Topic` type is actually synonym of `HexString32Bytes`. So this change is fine.